featuretools/__init__.py

```diff
@@ -1,13 +1,14 @@
-import config
-from .core import *
-import variable_types
-from .entityset.api import *
-from . import primitives
-from .synthesis.api import *
-from .primitives import Feature
-from .computational_backends.api import *
-from . import tests
-from .utils.pickle_utils import *
-import featuretools.demo
+import config  # noqa
```

Comment:
I believe you can put `# flake8: noqa` at the top of the file rather than repeat on every line.

Reply:
sounds good!